app/login.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="icon" href="favicon.png">
+    <link rel="stylesheet" href="css/main.css">
+    <title>LogoAuto</title>
+</head>
+
+<body>
+    <header class="header">
```

Comment:
I don't like the way elements are horizontally aligned 
fix it please
![Screenshot from 2021-07-04 17-46-22](https://user-images.githubusercontent.com/19241160/124389458-d728c580-dcef-11eb-8aff-9c0627c971eb.png)

Reply:
they should be on the same baseline